modules/op-configuring-basic-authentication-for-git.adoc

```diff
@@ -0,0 +1,84 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-configuring-basic-authentication-for-git_{context}"]
+= Configuring basic authentication for Git
+
+For basic authentication, {pipelines-shortname} creates a `~/.gitconfig` file containing the credentials specified in the secret, before executing any steps in a task run or a pipeline run. When the steps execute, {pipelines-shortname} uses those credentials to retrieve the pipeline resources specified in a run.
```

Comment:
Can we rephrase this so that the reader understands what they are going to do? The current style obscures what actions the user performs. 
For example (I'm not sure that this is technically accurate, but it is an example of the style I think might help.)
```suggestion
Before initiating a pipeline run, you configure basic authentication by creating a `~/.gitconfig` file that contains the credentials in the Git secret. {pipelines-shortname} uses those credentials to retrieve the pipeline resources specified in a run.
```

Reply:
IBM SG: "Use present tense. Use active voice. Use second person (you)."